src/main/cpp/events/sampleevent.cpp

```diff
@@ -462,62 +458,39 @@ void SampleEvent::mixBuffer( AudioBuffer* outputBuffer, int bufferPosition,
                     tgtBuffer[ i ] += (( s1 + ( s2 - s1 ) * frac ) * _volume );
                 }
             }
-            else if ( loopStarted && fi >= lo )
-            {
```

Comment:
Single conditional on _fBufferPointer_ assignment can remove this entire branch (which is mostly duplicate code anyways).